log/util.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import "fmt"
+
+// InterleavedKVToFields converts keyValues a la Span.LogKV() to a Field slice
+// a la Span.LogFields().
+func InterleavedKVToFields(keyValues ...interface{}) ([]Field, error) {
+	if len(keyValues)%2 != 0 {
+		return nil, fmt.Errorf("non-even keyValues len: %d", len(keyValues))
```

Comment:
haha :)

While I see what you are getting at (and while I recognize that this is micro-nit territory for both of us), I do like "non-even" because it is unmistakable. "Odd" has other (primary) connotations in English and would be easier to misinterpret.